src/TwilioVideo.ios.js

```diff
@@ -166,8 +170,8 @@ export default class TwilioVideo extends Component {
     this._eventEmitter = new NativeEventEmitter(TWVideoModule)
     
     this.setStereoEnabled = this.setStereoEnabled.bind(this)
-    // We expose this to the JS layer to allow avoiding the whole custom audio device code path via CodePush update if there is a critical bug
-    this.usesCustomAudioDevice = true;
+
+    this.useCustomAudioDevice = !!props.useCustomAudioDevice;
```

Comment:
Is the double negation necessary here if the `useCustomAudioDevice` is already a `boolean`?

Reply:
Yes. It's an optional property, so it could be undefined. `!!` ensures it always evaluates to true/false.